_topic_map.yml

```diff
@@ -697,6 +697,9 @@ Topics:
 - Name: Creating policy for Operator installations and upgrades
   File: olm-creating-policy
   Distros: openshift-origin,openshift-enterprise,openshift-webscale
+- Name: Managing admission webhooks
```

Comment:
Should this title be more specific that it's for Operators? Like how the actual assembly heading is "Managing admission webhooks in Operator Lifecycle Manager"?

Reply:
I was trying to keep the left-nav name shorter but I see now that I have used the "OLM" abbreviation in a left-nav already, so I am going to that again now!